bridge/client/app/_components/ktb-uniform-subscriptions/ktb-uniform-subscriptions.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '~@dynatrace/barista-components/core/src/style/variables';
+
+::ng-deep .subscription-list {
+  display: table;
+
+  > ktb-subscription-item {
+    display: table-row;
+
+    > * {
+      display: table-cell;
+    }
+  }
+}
+
+button[dt-button].disabled {
```

Comment:
If the button has the attribute `disabled` there should be no need for re-defining the css.